### DISCUSSION TITLE
How remove or control gap in bar stack

### DISCUSSION THREAD

**Discussion by mariusmjp** [IRIS-Solutions-Team/IRIS-Toolbox#407] [Q&A]
Hi

I want just to know if you have a way to remove empty space in stack bar graph.
For example I have value for period 0, 4 , 10, 100 and I want my graph just display the bar for this period with uniform space chose by me and don't show huge big gap between period in particularity between 10 and 100. Is it possible?

Thanks for your answer

**Comment by jaromir-benes** [comment]
Hi - this is a Matlab question, not exactly an Iris question. A short
answer is - no, if you want to preserve the dates on the x axis, there is
no way. Alternatively, you can simply extract your values into a plain
array, and plot them as such.

Best, Jaromir

**Reply by mariusmjp** [reply]
Thanks Jaromir for quick answer

For being more precise, I used rephrase.seriesChart(........., "Barmode", "stack", "Layout") and I was wondering if I could play with the layout options to get what I want.

**Comment by jaromir-benes** [comment]
Ok, now this is becoming a plotly.com issue - I refer you to the plotly
documentation for how to modify the layout to achieve what you wish... J
